src/App.jsx

```diff
@@ -7,15 +7,27 @@ function App() {
     const [todoList, setTodoList] = useState([]);
 
     const handleAddTodo = (newTodo) => {
-	setTodoList([...todoList, newTodo]);
+        const todoWithCompletion = {...newTodo, isCompleted:false};
```

Comment:
Well done implementing, adding a new property to your object then updating state.